# Plasmids, a Molecular Cornerston of Antimicrobial Resistance in the One Health Era

###  **Authors:** Salvador Castaneda-Barba, Eva M. Top, Thibault Stalder

## Link to publication

**********

**Abstract:** Antimicrobial resistance (AMR) poses a significant threat to human health. The widespread prevalence of AMR is largely due to the horizontal transfer of antibiotic resistance genes (ARG), typically mediated by plasmids. Many of the plasmid-mediated resistance genes in pathogens originate from environmental, animal, or human habitats. Despite evidence that plasmids mobilize ARG between these, we have a limited understanding of where ARG emerge and how their ecology and evolutionary trajectories result in them appearing in clinical pathogens. One Health, a holistic framework, enables the exploration of these knowledge gaps. With this in mind, we first provide an overview of how plasmids drive local and global AMR spread and link different habitats. Then we review some of the emerging studies integrating an eco-evolutionary perspective, opening a discussion on factors that affect the ecology and evolution of plasmids in complex microbial communities. Specifically, we discuss how the emergence and persistence of AMR plasmids can be affected by varying selective conditions, spatial structure, environmental heterogeneity, temporal variation, and coexistence with other members of the microbiome. These factors, along with others yet to be investigated, collectively determine the emergence and transfer of plasmid-mediated AMR within and between habitats at the local and global scale.  

**********

## Information about this repository:  

This repository contains the code for the meta-analysis carried out in the review paper titled "Plasmids, a Molecular Cornerstone of Antimicrobial Resistance in the One Health Era".  

### **File Description:**
#### **Data obtained from PLSDB**
The files named [plsdb.tsv]() and [plsdb.abr]() were downloaded from PLSDB, version 2021_06_23_v2. The file plsdb.tsv contains the meta-data for all the plasmids whilw the file plsdb.abr contains the resistance gene annotations. 
The files in the main directory named [Analysis.Rmd](https://github.com/marschmi/Diversity_Productivity/blob/master/Analysis.Rmd), [Analysis.html](Analysis.html), and [figures/](https://github.com/marschmi/Diversity_Productivity/tree/master/figures/) folder have all of the final analysis contents. **Please note that the [figures/](https://github.com/marschmi/Diversity_Productivity/tree/master/figures/) folder has all of the figures in the manuscript except Figure S6, which can be found [here](https://github.com/marschmi/Diversity_Productivity/blob/master/analysis/OTU_Removal_Analysis_Figs/figS6-1.png).**

Two supplementary analysis files can be found in the [analysis/](https://github.com/DenefLab/Diversity_Productivity/tree/master/analysis) folder. Specifically: Files named [OTU_Removal_Analysis.Rmd](https://github.com/marschmi/Diversity_Productivity/blob/master/analysis/OTU_Removal_Analysis.Rmd), and [OTU_Removal_Analysis.html](analysis/OTU_Removal_Analysis.html) are for the rare OTU sensitivity analysis. Files named [PrunedTree_Analysis.html](analysis/PrunedTree_Analysis.html) and [PrunedTree_Analysis.Rmd](https://github.com/DenefLab/Diversity_Productivity/blob/master/analysis/PrunedTree_Analysis.Rmd) are for different types of phylogenetic analyses. 
